eras/alonzo/impl/CHANGELOG.md

```diff
@@ -53,6 +53,7 @@
 
 ### `testlib`
 
+* Remove `Test.Cardano.Ledger.Alonzo.CostModel` in favor of `Test.Cardano.Ledger.Plutus`
```

Comment:
Maybe we should also mention the removal of `costModelParamsCount` from Arbitrary

Reply:
Done.